init/provision.sh

```diff
@@ -0,0 +1,7 @@
+echo ***PERFORMING DATABASE MIGRATIONS***
```

Comment:
@shinjonathan requesting to add steps to run the script.

Reply:
Not sure I understand. What step needs to be added?